pandas/_libs/algos.pyx

```diff
@@ -931,6 +931,32 @@ ctypedef fused rank_t:
     int64_t
 
 
+cdef rank_t get_rank_nan_fill_val(bint rank_nans_highest, rank_t[:] _=None):
```

Comment:
why is there an _ here?

Reply:
can rank_t be required

Reply:
The `_` was to signify that the argument is unused. Not requiring `rank_t` here is purposeful - the thought here was that we need an argument with `rank_t` present for fused type specialization, but with it being optional we never have to worry about passing anything.

Reply:
i c, might be worth comment ing in the doc-string as this is non-obvious (oh you did this nvm)